sqlx-runner/tx.go

```diff
@@ -35,6 +37,12 @@ func WrapSqlxTx(tx *sqlx.Tx) *Tx {
 	newtx := &Tx{Tx: tx, Queryable: &Queryable{tx}}
 	if dat.Strict {
 		time.AfterFunc(1*time.Minute, func() {
+			defer func() {
+				if r := recover(); r != nil {
+					fmt.Printf("recovered dat error: %s", r)
+				}
```

Comment:
Not sure what to put in here though...

I wonder if we should call `Rollback` here or something, but not sure if it will have any effect.

Reply:
what happens in this case? is the tx stuck in pending in postgres? @bruno-silva-cko

Reply:
That is what I want to have a look at today... but possibly it is already getting stuck because it is panicking I reckon. So it is not probably rolled back or cancelled anyways.